packages/html/src/treeview/templates/treeview-icons.tsx

```diff
@@ -0,0 +1,30 @@
+import { Treeview, TreeviewItem, TreeviewGroup } from "../../treeview";
+
+export const TreeviewIcons = (props) => (
+    <Treeview
+        children={[
+            <TreeviewItem text="Root 1" />,
```

Comment:
```suggestion
            <TreeviewItem text="Root 1" />
```

Reply:
The _TreeviewItem_ are defined as children, so I think we need the comma